cli/src/pcluster/resources/batch/docker/buildspec.yml

```diff
@@ -1,13 +1,13 @@
 version: 0.2
 
 phases:
-  install:
-    runtime-versions:
-      docker: 18
   pre_build:
     commands:
       - echo Logging in to Amazon ECR...
       - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+      - echo "Pulling amazonlinux:2 image from ECR"
+      - docker pull "public.ecr.aws/amazonlinux/amazonlinux:2"
```

Comment:
we could also just replace `FROM amazonlinux:2` in Dockerfile with `FROM public.ecr.aws/amazonlinux/amazonlinux:2`

Reply:
Done and retested